src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -39,12 +39,14 @@ public class ManagedCodeConventions
 
         private static readonly Dictionary<string, object> NetTFMTable = new Dictionary<string, object>
         {
-            { "tfm", new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.Net, FrameworkConstants.EmptyVersion) }
+            { "tfm", new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.Net, FrameworkConstants.EmptyVersion) },
+            { "tfm_raw", "net0" }
```

Comment:
It's not clear to me why the `tfm_raw` value is needed here. Doesn't the `TryGetValue` added to the pack rule mean this isn't needed?

Reply:
"just in case", honestly. It's not needed but I got paranoid.